mappings/net/minecraft/item/BlockPredicatesChecker.mapping

```diff
@@ -0,0 +1,8 @@
+CLASS net/minecraft/unmapped/C_uukjtqzj net/minecraft/item/BlockPredicatesChecker
+	FIELD f_ckvsugox lastCheckedBlock Lnet/minecraft/unmapped/C_gogfiawf;
+	FIELD f_fgfhnthg lastResult Z
+	FIELD f_mmiqnzjg tagName Ljava/lang/String;
```

Comment:
Can't look at the sources, but what kind of tag is this?

If it's NBT, then we don't use `tag`.